doc/install_matomo.rst

```diff
@@ -0,0 +1,96 @@
+Install Matomo
```

Comment:
Don't close this PR until after I commit the docs on installing docker-Matomo as well.